sim/Makefrag

```diff
@@ -67,6 +68,12 @@ $(verilator_debug): $(HEADER) $(DRIVER_CC) $(DRIVER_H) $(midas_cc) $(midas_h)
 verilator: $(verilator)
 verilator-debug: $(verilator_debug)
 
+run-verilator: $(verilator)
```

Comment:
How does this differ from the existing run commands?

Reply:
Also if we want to keep it, it should be added for vcs too

Reply:
This allows you to run arbitrary binaries, not just the ones that can be symlinked to output/. I was just basing it off the run-xsim rule.

Reply:
Okay, that sounds useful. Can you add it for vcs also? 

@davidbiancolin

Reply:
I don't have a firesim installation on the millennium mahcines set up for testing, can you or David test the commit I just pushed?